src/daft-dsl/src/functions/kv/kv_functions.rs

```diff
@@ -0,0 +1,331 @@
+use common_error::{DaftError, DaftResult};
+use daft_core::series::Series;
+use daft_schema::schema::Schema;
+#[cfg(feature = "python")]
+use pyo3::Python;
+#[cfg(feature = "python")]
+use pyo3::types::PyAnyMethods;
+use serde::{Deserialize, Serialize};
+
+use crate::functions::{ExprRef, FunctionArgs, ScalarUDF};
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash)]
+pub struct KVGetWithStoreName;
+
+#[typetag::serde]
+impl ScalarUDF for KVGetWithStoreName {
+    fn name(&self) -> &'static str {
+        "kv_get_with_name"
+    }
+
+    fn call(&self, args: FunctionArgs<Series>) -> DaftResult<Series> {
+        let inputs = args.into_inner();
```

Comment:
would prefer to use structured args instead of `into_inner()`. many of the other scalarUDF impls have examples of using the structured arguments. 


Same comment applies to other udf impls in this file too.